index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+    <header>
+        <h2><a href="#">Welcome</a></h2>
+        <i class="fa fa-bars" aria-hidden="true"></i>
```

Comment:
- [ ] You've done really well in implementing the toolbar. However, as per the requirements, the hamburger button have a `nav` tag.
![image](https://user-images.githubusercontent.com/18730106/143309674-42474bb4-92dd-4475-b253-5537aeaf6bca.png)
Something like this would be nice. 
```suggestion
        <nav>
          <i class="fa fa-bars" aria-hidden="true"></i>
        </nav>
```